src/main/java/core/basesyntax/CalculateArea.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface CalculateArea {
```

Comment:
let's remove this interface
you have interface AreaCalculator

Reply:
removed